packages/flutter/lib/src/material/bottom_navigation_bar.dart

```diff
@@ -605,7 +605,11 @@ class _Tile extends StatelessWidget {
         heightFactor: 1,
         child: Row(
           mainAxisSize: MainAxisSize.min,
-          children: <Widget>[icon, const SizedBox(width: 8), label],
+          children: <Widget>[
+            icon,
+            const SizedBox(width: 8),
+            Flexible(child: IntrinsicWidth(child: label))
```

Comment:
A comment that explains what this unusual combination of widgets is for would be helpful here.

Reply:
Okay. So defining what `Flexible` and `IntrinsicWidth` does for the `label` would be fine, right?

Reply:
It's pretty obvious what `<Widget>[icon, const SizedBox(width: 8), label]` means however it's not obvious what wrapping `label` in Flexible and IntrinsicWidth widgets is intended to accomplish in the new version. Which is to say, "yes" :-)